stamina-testkit/src/test/scala/stamina/testkit/ScalatestTestGenerationSpec.scala

```diff
@@ -11,48 +11,71 @@ class ScalatestTestGenerationSpec extends StaminaTestKitSpec {
 
   import TestDomain._
 
-  case class ItemPersister(override val key: String) extends Persister[Item, V1](key) {
+  case class ItemPersister[V <: Version: VersionInfo](override val key: String) extends Persister[Item, V](key) {
```

Comment:
I am not sure if I understand correctly. Are you talking about making `ItemPersister` more sophisticated and make different `unpersist` method behaviour for different `Persisted.version` values?